www/js/lib/zimArchive.js

```diff
@@ -33,10 +33,17 @@ import utf8 from './utf8.js';
 /**
  * ZIM Archive
  *
- *
  * @typedef ZIMArchive
- * @property {ZIMFile} _file The ZIM file (instance of ZIMFile, that might physically be split into several actual files)
- * @property {String} _language Language of the content
```

Comment:
I felt it was unnecessary to mark all of these metadata, including the abstraction of the ZIMFile, as private variables. So, I've changed them to normal (removed the starting _).

However, I agree that ZIMArchive.file._files (the _files part) should remain as a private variable, but that is defined in ZIMFile.